src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -820,29 +820,41 @@ private static void clip3primeEndsTo5primeEnds(final SAMRecord pos, final SAMRec
           For the positive strand, we ask for the position of the 3' most base which will not be clipped, and then increment to find the 5' most base to clip.
           We do this because getReadPositionAtReferencePositionIgnoreSoftClips will return the base before a deletion, so will return the 3' most base before
           the queried base when the queried base is in a deletion on a positive strand read
+
+            3'<SSSSSSSSMMMMMMMMMMMM5'
```

Comment:
the 5 looks like an S...
```suggestion
            3' <SSSSSSSSMMMMMMMMMMMM 5'
```